mava/systems/ippo/networks.py

```diff
@@ -236,14 +266,17 @@ def policy_fn(inputs: jnp.ndarray) -> networks_lib.FeedForwardNetwork:
             # Add optional feedforward layers after the recurrent layers
             hk.nets.MLP(
                 policy_layers_after_recurrent,
-                activation=jax.nn.relu,
+                activation=activation_function,
+                w_init=w_init_fn(orthogonal_initialisation, jnp.sqrt(2)),
                 activate_final=True,
             ),
 
         # Add a categorical value head.
         policy_network.append(
             networks_lib.CategoricalHead(
-                num_values=num_actions, dtype=environment_spec.actions.dtype
+                num_values=num_actions,
+                dtype=environment_spec.actions.dtype,
+                w_init=w_init_fn(orthogonal_initialisation, 0.01),
```

Comment:
@RuanJohn  The network layer gain is exactly what you implemented here. I checked their [implementation](https://github.com/marlbenchmark/on-policy/blob/main/onpolicy/algorithms/utils/act.py#L32) and this is exactly what they are doing. So we already have this covered. What I may suggest is to make the gain value a parameter so that we can play around with the value. In the paper they have experiments with values of 0.01 and 1. We can link this [issue](https://github.com/instadeepai/Mava/issues/789) to this PR. Also it will be great if we had a wrapper for the initialisation so that we can easily try different ones. Not just have orthogonal_initialisation as the only choice along with the default.

Reply:
Thank you for looking into this @ulricharmel. I have added the policy gain as a parameter to the networks. I agree that we can increase the flexibility in the future but I think we can do this as required or expand functionality after the release since we already have an idea of what scaling values to use. Please let me know your thoughts?